python-api/products_ranker_api/spiders/aliexpress_spider.py

```
@@ -0,0 +1,57 @@
+    def start_requests(self):
+        formatted_query = self.query.replace(" ", "+")
+        url = f"https://www.aliexpress.com/wholesale?SearchText={formatted_query}"
+        yield scrapy.Request(url, callback=self.parse)
+
+    def parse(self, response, **kwargs):
+        now = datetime.utcnow().isoformat()
+        products = []
+
+        for item in response.css(".manhattan--container--1lP57Ag"):  # This class might change
+            name = item.css("a::attr(title)").get()
+            link = item.css("a::attr(href)").get()
```

Ruff (0.8.2)

Local variable `link` is assigned to but never used

Remove assignment to unused variable `link`

(F841)